spikeinterface/sorters/runsorter.py

```diff
@@ -246,7 +265,7 @@ def run_sorter_container(sorter_name, recording, mode, container_image, output_f
 
 # run in docker
 output_folder = '{output_folder}'
-run_sorter_local('{sorter_name}', recording, output_folder=output_folder,
+run_sorter_local('{sorter_name}', recording, gpu_to_use={0 if gpu_to_use else None}, output_folder=output_folder,
```

Comment:
Should this be one?
```suggestion
run_sorter_local('{sorter_name}', recording, gpu_to_use={1 if gpu_to_use else None}, output_folder=output_folder,
```

Reply:
I realize that I should make a comment on why it is this way. Part in new commit:
```
    if gpu_to_use is None:
        gpu_to_use_container = None
    # The GPU indexing inside the container starts from 0
    elif isinstance(gpu_to_use, int):
        # in cases of single GPU passthrough, the index is zero
        gpu_to_use_container = [0]
    else:
        # in cases where we pass several GPUs 0, 1, ..., n GPUs
        gpu_to_use_container = list(range(len(gpu_to_use)))
```